### ISSUE TITLE
Should the check reporter write the result object to a file?

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/testthat#795]
So that it can be inspected post mortem? Cf. https://github.com/r-lib/rcmdcheck/issues/67

**Comment by dschlaep** [comment]
Thumbs up! My situation is that `devtools::test()`, `devtools::check()`, `R CMD check .` etc are passing on a range of local machines, but not on CIs that runs `R CMD check` -- debugging is really tricky without access to the full check reporter output. Thus, it would be very helpful if I could instruct check reporter to write full output to a file. Thanks!

**Comment by hadley** [comment]
Doesn't R already create a file? 

If we need to create one, what should we call it so that travis automatically displays? (cc @jimhester)

**Comment by gaborcsardi** [comment]
I was more thinking about an RDS with R objects that you can inspect later. E.g. sg like `dump.frames()`. With some limit on size somehow, but e.g. on Travis it does not even matter if the file is big.

**Comment by hadley** [comment]
Oh then is this a duplicate of #658?

**Comment by gaborcsardi** [comment]
Possibly.

**Comment by hadley** [comment]
I'm going to close this issue, and we can revisit, if needed, once #658 is done.
